Add a draft covid postponement annoucnement (#1223)
writethedocs/www: docs/conf/prague/2020/news/prg-covid-19-update-02.rst

Hello everyone,

> Since our :doc:`first announcement </conf/prague/2020/news/prg-covid-19-update-01>` two weeks ago, we've been following the COVID-19 updates from the relevant authorities. At the time of writing this update, Czech Republic is still under lockdown and the government predicts that this situation will continue for at least a month.

Reviewer: Not precisely true, https://www.unian.info/world/covid-19-worldwide-czech-republic-opens-borders-10971311.html
https://www.reuters.com/article/us-health-coronavirus-czech/czechs-lift-ban-on-movement-and-travel-speed-up-reopening-shops-as-new-covid-19-cases-slow-idUSKCN22531H

```suggestion
Since our :doc:`first announcement </conf/prague/2020/news/prg-covid-19-update-01>` two weeks ago, we've been following the COVID-19 updates from the relevant authorities. At the time of writing this update, Czech Republic is still under quarrantine.
```

Author: I believe this is just updating the URL in the old post. We don't want ot change the actual content.

> Given the uncertainty of the long-term impact of this situation, we have decided to **postpone the Call for Proposals and ticket sales until at least 1st May, 2020**. We hope that by then we will have more information to determine whether to proceed with the conference on the original dates, postpone the conference, or cancel the conference altogether.

Reviewer: Should "1st May, 2020" be updated?
